client/src/components/dashboard/deploy/DeployPage.js

```diff
@@ -11,7 +12,6 @@ import './DeployPage.css';
 const DeployPage = ({ teamId }) => {
   const dispatch = useDispatch();
   const loading = useSelector((state) => state.deploy.loading);
-  // TODO: refactor this into useState
```

Comment:
So we're not going with `useState`?

Reply:
cause before we wanna check if the user has added any page before deploy, but now we made `client homepage` mandatory, so I think we don't need `useState` atm